shared/redux/rtk-apis/apiSlice.ts

```diff
@@ -8,8 +8,16 @@ export const apiSlice = createApi({
     endpoints: (builder) => ({
         getTasks: builder.query<ITask[], void>({
             query: () => '/tasks',
-        })
+        }),
+
+        createTask: builder.mutation<ITask, Partial<ITask>>({
```

Comment:
```suggestion
        createTask: builder.mutation<ITask, TCreateTaskDto>({
```
Might have to create a new type. Best not to bypass type specificity by usage of Partials.